contracts/Registry.sol

```diff
@@ -45,10 +50,66 @@ contract TablelandTables is
         setBaseURI(baseURI);
     }
 
-    event RunSQL(uint256 tableId, address caller, string statement);
+    event CreateTable(address caller, uint256 tableId, string statement);
+
+    function createTable(address caller, string memory statement) public {
+        uint256 tokenId = _tokenIdCounter.current();
+        _tokenIdCounter.increment();
+        _safeMint(caller, tokenId);
+
+        emit CreateTable(caller, tokenId, statement);
+    }
+
+    event RunSQL(
+        address caller,
+        uint256 tokenId,
+        string statement,
```

Comment:
The `isOwner` property is missing. The validator needs that to decide if grants are allowed or not.

Reply:
added